firebase-firestore/firebase-firestore.gradle

```diff
@@ -136,6 +142,9 @@ dependencies {
 
     //To provide @Generated annotations
     compileOnly 'javax.annotation:jsr250-api:1.0'
+    //To provide Kotlin @SerialInfo support
+    compileOnly 'org.jetbrains.kotlinx:kotlinx-serialization-core:1.3.2'
+    compileOnly "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
```

Comment:
We need dontwarns for these in firestore's proguard file

Reply:
Thank you, I have added these two to the proguard file.